working-groups.md

```diff
@@ -93,7 +93,7 @@ This is the [`tektoncd/triggers`](https://github.com/tektoncd/triggers) working
 | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------                                                                             |
 | Forum                      | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev)                                                                                                                                                                         |
 | Community Meeting VC       | [https://meet.google.com/oqm-imjh-uzu](https://meet.google.com/oqm-imjh-uzu)                                                                                                                                                              |
-| Community Meeting Calendar | Tuesdays every week, 12:00p-12:30p PST <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=ZmNmbWtnNGxzcG1uaGUwNm9rMHB1MzR2ZnJfMjAyMDAxMjhUMTcwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL) |
+| Community Meeting Calendar | Wednesdays every week, 11:00p-11:30p EST <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=ZmNmbWtnNGxzcG1uaGUwNm9rMHB1MzR2ZnJfMjAyMDAxMjhUMTcwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL) |
```

Comment:
https://calendar.google.com/event?action=TEMPLATE&tmeid=ZDA3YzZldXZqZ2M1ajBndWVkNzRlNTdzN2ZfMjAyMDExMThUMTYwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL

i think the calendar link might need an update too - i dont really understand how it works but the current link seems to want to add a repeating even on tuesdays, the one above starts this week on wednesdays

(maybe they both work? not sure!)

Reply:
done!